pkg/controller/provider/container/openstack/model.go

```diff
@@ -741,7 +741,7 @@ func (r *VolumeAdapter) GetUpdates(ctx *Context) (updates []Updater, err error)
 
 				volume.ApplyTo(m)
 				err = tx.Update(m)
-				if err == nil {
+				if err != nil {
```

Comment:
shouldn't it be `err == nil`?

Reply:
yeah... I changed it to see if it reproduces without the fix and forgot to remove before the second commit

Reply:
ah ok

Reply:
and did reproduce? :)

Reply:
yeah 😅